java/ql/test/TestUtilities/InlineExpectationsTest.qll

```diff
@@ -202,6 +202,7 @@ private predicate getAnExpectation(
       column = TDefaultColumn() and
       exists(int end |
         end = getEndOfColumnPosition(0, content) and
+        not exists(string name | exists(TNamedColumn(name)) and content.indexOf(name + ":") < end) and
```

Comment:
I haven't looked too far into the machinery of `InlineExpectationsTest.qll`, but did this turn out to be a problem for your test? I couldn't see anything with `MISSING:` in it :confused: so if you have a motivating example, that would be great :+1:

Reply:
Sure, here's an example that produces spurious missing-value warnings at every use of `// $ MISSING:`:

```java
public class Test {

  public static void main1() {} // $MISSING: abc
  public static void main2() {} //$MISSING:abc
  public static void main3() {} //$MISSING: abc
  public static void main4() {} // $MISSING:abc

} // $MISSING: abc
```

Reply:
Oh, I see this has been fixed by https://github.com/github/codeql/pull/6096 now :+1: I'm not sure if you still want this change to go in for the inline-tests? (this is more in the category of nitpicks I guess)

Reply:
This line is now superfluous and the fix commit on this PR can be removed.

Reply:
:+1: rebased and dropped

Reply:
awesome, thanks :heart: